src/components/pages/index.ts

```diff
@@ -1,7 +1,6 @@
 import About from './About/About';
-import Create from './Create/Create';
 import Home from './Home/Home';
 import Manage from './Manage/Manage';
 import NotFound from './NotFound/NotFound';
 
-export { About, Create, Home, Manage, NotFound };
+export { About, Home, Manage, NotFound };
```

Comment:
removing the `Create` page entirely?

Reply:
Well, we discussed the create flow being a modal, accessed by the same buttons that link to the create page, and we no longer have designs for a specific create ant page (because those designs are now a for the modal) - so i figured yeah remove it for now.